editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -24,17 +24,20 @@ import {
  */
 import { replaceBlock } from '../../actions';
 
-function InvalidBlockWarning( { ignoreInvalid, switchToDefaultType } ) {
+function InvalidBlockWarning( { ignoreInvalid, switchToBlockType } ) {
+	const htmlBlockName = 'core/html'; // hard-coded by maybe replace by an API similar to getUnknownTypeHandlerName
```

Comment:
Should we add another `getHtmlBlockName` API? or maybe we don't mind hardcoding this special case

Reply:
I don't think I mind doing `core/html`.

Reply:
Could we remove the comment or convert it to having a `TODO` prefix with whatever we decide here? I would probably be fine with explicitly setting `core/html` as well, particularly because we're still testing that the desired block type is registered before presenting it as an option.